mac/KeymanEngine4Mac/KeymanEngine4MacTests/KMEngineTests.m

```diff
@@ -0,0 +1,99 @@
+//
+//  KMEngineTests.m
+//  KeymanEngine4MacTests
+//
+//  Created by tom on 1/18/18.
+//  Copyright © 2018 SIL International. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <Carbon/Carbon.h>
+#import "KeymanEngineTestsStaticHelperMethods.h"
+#import "KMEngine.h"
+
+@interface KMEngineTests : XCTestCase
+
+@end
+
+@implementation KMEngineTests
+
+- (void)setUp {
+    [super setUp];
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
```

Comment:
What are the preconditions for running these tests? Will they run without Keyman app being installed (e.g. on a build agent)?

Reply:
They shouldn't require anything to be installed.